### PR TITLE
[new-package] ThorVG 0.12.10

### PR DESCRIPTION
**Pull request by vtorri**

### CODE REVIEW

File: mingw-w64-thorvg/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+_realname=thorvg
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=0.12.8
+pkgrel=1
+pkgdesc="Lightweight vector graphic library (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url='https://www.thorvg.org/'
+license=('LICENSE')
```
Comment (Biswa96) [inline mingw-w64-thorvg/PKGBUILD:12]:
```suggestion
license=('spdx:MIT')
```

File: mingw-w64-thorvg/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+makedepends=("${MINGW_PACKAGE_PREFIX}-meson"
+             "${MINGW_PACKAGE_PREFIX}-ninja"
+             "${MINGW_PACKAGE_PREFIX}-pkg-config"
+             "${MINGW_PACKAGE_PREFIX}-cc"
+             "${MINGW_PACKAGE_PREFIX}-libpng"
```
Comment (Biswa96) [inline mingw-w64-thorvg/PKGBUILD:17]: These three libraries should be in `depends=()` list.

Comment (vtorri) [reply mingw-w64-thorvg/PKGBUILD:17]: hmm, glib's PKGBUILD should be fixed then

Comment (vtorri) [reply mingw-w64-thorvg/PKGBUILD:17]: thre libraries ? you mean libpng, libjpeg and libwebp ?

Comment (Biswa96) [reply mingw-w64-thorvg/PKGBUILD:17]: I meant that libpng, libjpeg-turbo, libwebp should be in `depends=()` list.

**Pull request merged**